source/main/terrain/TerrainManager.h

```diff
@@ -74,39 +74,29 @@ class TerrainManager : public ZeroedMemoryAllocator
 
 protected:
 
-    RoR::ConfigFile m_terrain_config;
-
     // subsystems
     Character* character;
     Collisions* collisions;
     Dashboard* dashboard;
     Envmap* envmap;
     HDRListener* hdr_listener;
-    SurveyMapManager* survey_map;
+    SurveyMapManager* m_survey_map;
     ShadowManager* shadow_manager;
     SkyManager* sky_manager;
     TerrainGeometryManager* geometry_manager;

```

Comment:
@AnotherFoxGuy There are 4 duplicates of this here. Can you tweak the review tool?